dbMerger/storer/fullDBMerger_test.go

```diff
@@ -201,5 +208,6 @@ func TestDataMerger_MergeDBs(t *testing.T) {
 		assert.True(t, copyCalled)
 		assert.Equal(t, 3, numPersistersCreated)
 		assert.True(t, mergeDBCalled)
+		assert.Equal(t, 2, numClosedPersisters) // 3 sources, 1 copied, 2 opened to copy key by key
```

Comment:
From the comment, one reads that `3 sources - 1 copied == 2 opened`, but this understanding is not correct (I think). Perhaps update the comment to be a little more explicit, or record the path / name of the closed persisters and assert by that name (instead of using a counter, which loses that information).

Reply:
actually is ok, 2 were actually opened. 1 was copied as an OS copy operation so it did not involved opening a DB or something.